doc/start/os-recommendations.rst

```diff
@@ -43,7 +43,7 @@ package (i.e., sysvinit, systemd).
 +--------------+--------+------------------------+--------------------------------+-------------------+-----------------+
 | Release Name | Tag    | CentOS                 | Ubuntu                         | OpenSUSE :sup:`C` | Debian :sup:`C` |
 +==============+========+========================+================================+===================+=================+
-| Quincy       | 17.2.z | 8 :sup:`A`             | 20.04 :sup:`A`                 | 15.3              | 11              |
+| Quincy       | 17.2.z | 8 :sup:`A` 9 :sup:`1`  | 20.04 :sup:`A`                 | 15.3              | 11              |
```

Comment:
It looks like centos 8 is still mentioned?  The way I'd interpret
```
8 :sup:`A` 9 :sup:`1`
```
is "8 has been and continues to be A and something is added for 9".  That is not a message that we trying to send.